index.html

```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+	      
+	<div class="card">
+          <p class="name">Timmothy Rain</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/rRubedog" target="_blank">@rRubedog</a>
+          </p>
+          <p class="about">
+			I am currently taking a programing class in high school and I am working to becoming a contributing programer/web dev in the future.
+		  	I highly enjoy coding websites with HTML, CSS, and JS so I am planing on becoming a front-end web dev.
+			Since I have a really weird music taste I am currently making a website for normal people to learn about the music that I listen to, it was originally being made for school but I missed it so I'm going to try my best to make it as reliable as possible, you can check that out <a href="https://github.com/rRubedog/EDM-Hardcore-Genres"><b>here</b></a>.
```

Comment:
```suggestion
			Since I have a really weird music taste I am currently making a website for normal people to learn about the music that I listen to, it was originally being made for school but I missed it so I'm going to try my best to make it as reliable as possible, you can check that out <a href="https://github.com/rRubedog/EDM-Hardcore-Genres" target="_blank"><b>here</b></a>.
```